src/js/menu/menu-button.js

```diff
@@ -70,11 +70,13 @@ class MenuButton extends ClickableComponent {
 
     // Add a title list item to the top
     if (this.options_.title) {
-      menu.contentEl().appendChild(Dom.createEl('li', {
+      let title = Dom.createEl('li', {
```

Comment:
The menu title should probably be a separate dive inside menu with the menu items being in a separate UL, but we can't make the change in a minor update.